env/ORION.env

```diff
@@ -56,7 +56,7 @@ elif [[ "${step}" = "prep_emissions" ]]; then
     export APRUN="${launcher} -n 1"
 
 elif [[ "${step}" = "waveinit" ]] || [[ "${step}" = "waveprep" ]] || [[ "${step}" = "wavepostsbs" ]] || \
-    [[ "${step}" = "wavepostbndpnt" ]] || [[ "${step}" = "wavepostpnt" ]] || [[ "${step}" == "wavepostbndpntbll" ]]; then
+    [[ "${step}" = "wavepostbndpnt" ]] || [[ "${step}" = "wavepostpnt" ]] || [[ "${step}" == "wavepostbndpntbll" ]] || [[ "${step}" = "wave_stat" ]]; then
```

Comment:
I don't see the `wave_stat_pnt` job added in any of the env files.

Reply:
I think we can not have that here if there are no USE_CFP, etc needed. I also had the same thought as Kate though.